resources/web/wwi/x3d.js

```diff
@@ -823,15 +829,26 @@ THREE.X3DLoader = class X3DLoader {
       return;
 
     var attenuation = convertStringToVec3(getNodeAttribute(light, 'attenuation', '1 0 0'));
-    var color = convertStringTorgb(getNodeAttribute(light, 'color', '1 1 1'));
+    var color = convertStringToColor(getNodeAttribute(light, 'color', '1 1 1'), false);
     var intensity = parseFloat(getNodeAttribute(light, 'intensity', '1'));
     var location = convertStringToVec3(getNodeAttribute(light, 'location', '0 0 0'));
     var radius = parseFloat(getNodeAttribute(light, 'radius', '100'));
     var castShadows = getNodeAttribute(light, 'castShadows', 'false').toLowerCase() === 'true';
 
-    var lightObject = new THREE.PointLight(color.getHex(), intensity);
-    lightObject.decay = attenuation.x;
+    var lightObject = new THREE.PointLight(color.getHex());
+
+    // Tradeoff to let cohabit VRML light attenuation and the threejs light "physically correct mode".
```

Comment:
we should be consistent in the way we write three.js. In another file we use `three.js`.

I propose to change it here, but it is also fine to change the other occurrence(s).

```suggestion
    // Tradeoff to let cohabit VRML light attenuation and the three.js light "physically correct mode".
```

Reply:
We should certainly use `three.js`, this is the terminology used on their doc: https://threejs.org/docs/index.html#manual/en/introduction/Creating-a-scene